functional-tests/tests/test_anonymizer.py

```diff
@@ -24,7 +24,7 @@ def test_given_anonymize_called_with_valid_request_then_expected_valid_response_
 
     response_status, response_content = anonymize(request_body)
 
-    expected_response = '{"text":"hello world, my name is ANONYMIZED. ' \
```

Comment:
You can also use the json equality method, more deterministic.
Also I would align with the other methods defining string in triple quotes, and drop the \n